### ISSUE TITLE
CI incorrectly reporting no matching files

### ISSUE THREAD

**Issue by MattIPv4** [cdnjs/packages#259]
As seen in the CI run on https://github.com/cdnjs/packages/pull/190, it is failing the CI run reporting that no files match the given auto-update config. However, if you read the full CI run report it is indicating it matches the correct number of files for the most recent version.

```
No files will be published for version 7.4.6.
```

```
 10 last versions:
- 7.4.6: 4 file(s) matched :heavy_check_mark:
- 7.4.5: 4 file(s) matched :heavy_check_mark:
- 7.4.4: 0 file(s) matched :heavy_exclamation_mark:
- 7.4.3: 0 file(s) matched :heavy_exclamation_mark:
- 7.4.2: 0 file(s) matched :heavy_exclamation_mark:
- 7.4.1: 0 file(s) matched :heavy_exclamation_mark:
- 7.4.0: 0 file(s) matched :heavy_exclamation_mark:
- 7.3.1: 0 file(s) matched :heavy_exclamation_mark:
- 7.3.0: 0 file(s) matched :heavy_exclamation_mark:
- 7.2.1: 0 file(s) matched :heavy_exclamation_mark:
```

**Comment by klausenbusk** [comment]
`checker` isn't doing a `checkout` initially it seems (looking into this now):
```
Running in debug mode
packages/packages/w/wikibase-sdk.json: /tmp/git013345953: run /usr/bin/git clone git://github.com/maxlath/wikibase-sdk.git .
packages/packages/w/wikibase-sdk.json: run /usr/bin/git tag
packages/packages/w/wikibase-sdk.json: found tags in git: [v0.0.2 v0.0.3 v0.0.4 v0.0.5 v0.0.6 v0.1.0 v0.1.1 v1.0.0 v1.0.1 v1.0.2 v1.1.0 v1.2.0 v1.2.1 v1.3.0 v1.3.1 v1.3.2 v1.3.3 v1.3.4 v1.3.5 v1.3.6 v1.3.7 v1.4.0 v1.4.1 v1.4.2 v1.4.3 v1.4.4 v2.0.0 v2.1.0 v2.1.1 v2.2.0 v2.2.1 v2.2.2 v2.3.0 v2.4.0 v2.4.1 v2.5.0 v2.6.0 v3.0.0 v3.0.1 v3.1.0 v3.2.0 v3.2.1 v3.2.2 v3.2.3 v3.2.4 v4.0.0 v4.0.1 v4.0.2 v4.0.3 v4.0.4 v4.0.5 v4.0.6 v4.1.0 v4.2.0 v4.2.1 v4.2.2 v4.2.3 v4.2.4 v4.2.5 v4.3.0 v4.3.1 v4.3.2 v4.3.3 v5.0.0 v5.1.0 v5.1.1 v5.1.2 v5.1.3 v5.1.4 v5.10.0 v5.10.1 v5.11.0 v5.11.1 v5.11.2 v5.12.0 v5.13.0 v5.13.1 v5.14.0 v5.15.0 v5.15.1 v5.15.10 v5.15.11 v5.15.2 v5.15.3 v5.15.4 v5.15.5 v5.15.6 v5.15.7 v5.15.8 v5.15.9 v5.16.0 v5.16.1 v5.16.2 v5.2.0 v5.2.1 v5.2.2 v5.2.3 v5.2.4 v5.2.5 v5.2.6 v5.2.7 v5.2.8 v5.2.9 v5.3.0 v5.3.1 v5.4.0 v5.5.0 v5.6.0 v5.6.1 v5.6.2 v5.6.3 v5.7.0 v5.7.1 v5.7.2 v5.7.3 v5.7.4 v5.7.5 v5.7.6 v5.8.0 v5.8.1 v5.9.0 v5.9.1 v5.9.2 v5.9.3 v5.9.4 v5.9.5 v5.9.6 v6.0.0 v6.0.1 v6.1.0 v6.2.0 v6.3.0 v6.4.0 v6.5.0 v6.6.0 v6.7.0 v7.0.0 v7.0.1 v7.0.10 v7.0.11 v7.0.12 v7.0.13 v7.0.2 v7.0.3 v7.0.4 v7.0.5 v7.0.6 v7.0.7 v7.0.8 v7.0.9 v7.1.0 v7.1.1 v7.1.2 v7.1.3 v7.1.4 v7.1.5 v7.1.6 v7.2.0 v7.2.1 v7.3.0 v7.3.1 v7.4.0 v7.4.1 v7.4.2 v7.4.3 v7.4.4 v7.4.5 v7.4.6 v7.5.0 v7.6.0 v7.6.1 v7.6.2]
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v0.0.2
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v0.0.3
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v0.0.4
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v0.0.5
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v0.0.6
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v0.1.0
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v0.1.1
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v1.0.0
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v1.0.1
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v1.0.2
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v1.1.0
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v1.2.0
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v1.2.1
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v1.3.0
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v1.3.1
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v1.3.2
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v1.3.3
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v1.3.4
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v1.3.5
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v1.3.6
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v1.3.7
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v1.4.0
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v1.4.1
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v1.4.2
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v1.4.3
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v1.4.4
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v2.0.0
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v2.1.0
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v2.1.1
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v2.2.0
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v2.2.1
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v2.2.2
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v2.3.0
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v2.4.0
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v2.4.1
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v2.5.0
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v2.6.0
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v3.0.0
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v3.0.1
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v3.1.0
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v3.2.0
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v3.2.1
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v3.2.2
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v3.2.3
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v3.2.4
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v4.0.0
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v4.0.1
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v4.0.2
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v4.0.3
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v4.0.4
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v4.0.5
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v4.0.6
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v4.1.0
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v4.2.0
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v4.2.1
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v4.2.2
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v4.2.3
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v4.2.4
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v4.2.5
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v4.3.0
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v4.3.1
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v4.3.2
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v4.3.3
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v5.0.0
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v5.1.0
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v5.1.1
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v5.1.2
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v5.1.3
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v5.1.4
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v5.10.0
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v5.10.1
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v5.11.0
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v5.11.1
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v5.11.2
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v5.12.0
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v5.13.0
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v5.13.1
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v5.14.0
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v5.15.0
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v5.15.1
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v5.15.10
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v5.15.11
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v5.15.2
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v5.15.3
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v5.15.4
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v5.15.5
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v5.15.6
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v5.15.7
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v5.15.8
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v5.15.9
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v5.16.0
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v5.16.1
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v5.16.2
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v5.2.0
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v5.2.1
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v5.2.2
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v5.2.3
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v5.2.4
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v5.2.5
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v5.2.6
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v5.2.7
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v5.2.8
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v5.2.9
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v5.3.0
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v5.3.1
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v5.4.0
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v5.5.0
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v5.6.0
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v5.6.1
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v5.6.2
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v5.6.3
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v5.7.0
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v5.7.1
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v5.7.2
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v5.7.3
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v5.7.4
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v5.7.5
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v5.7.6
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v5.8.0
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v5.8.1
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v5.9.0
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v5.9.1
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v5.9.2
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v5.9.3
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v5.9.4
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v5.9.5
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v5.9.6
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v6.0.0
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v6.0.1
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v6.1.0
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v6.2.0
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v6.3.0
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v6.4.0
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v6.5.0
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v6.6.0
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v6.7.0
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v7.0.0
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v7.0.1
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v7.0.10
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v7.0.11
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v7.0.12
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v7.0.13
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v7.0.2
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v7.0.3
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v7.0.4
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v7.0.5
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v7.0.6
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v7.0.7
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v7.0.8
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v7.0.9
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v7.1.0
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v7.1.1
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v7.1.2
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v7.1.3
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v7.1.4
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v7.1.5
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v7.1.6
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v7.2.0
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v7.2.1
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v7.3.0
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v7.3.1
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v7.4.0
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v7.4.1
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v7.4.2
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v7.4.3
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v7.4.4
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v7.4.5
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v7.4.6
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v7.5.0
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v7.6.0
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v7.6.1
packages/packages/w/wikibase-sdk.json: run /usr/bin/git log -1 --format=%aI v7.6.2

most recent version: 7.6.2
packages/packages/w/wikibase-sdk.json: match *.js in /tmp/git013345953/dist but doesn't exists
::error file=packages/packages/w/wikibase-sdk.json,line=1,col=1::No files will be published for version 7.6.2.%0A[Click here to debug your glob pattern `*.js`](https://www.digitalocean.com/community/tools/glob?comments=true&glob=%25!A(MISSING).js&matches=true&tests=.eslintrc&tests=HEAD&tests=config&tests=description&tests=applypatch-msg.sample&tests=commit-msg.sample&tests=fsmonitor-watchman.sample&tests=post-update.sample&tests=pre-applypatch.sample&tests=pre-commit.sample&tests=pre-merge-commit.sample&tests=pre-push.sample&tests=pre-rebase.sample&tests=pre-receive.sample&tests=prepare-commit-msg.sample&tests=update.sample&tests=index&tests=exclude&tests=HEAD&tests=master&tests=HEAD&tests=pack-f0c917a7e7a5538b5a7d2ab1036f4eb3ec4d256c.idx&tests=pack-f0c917a7e7a5538b5a7d2ab1036f4eb3ec4d256c.pack&tests=packed-refs&tests=master&tests=HEAD&tests=pre-commit&tests=.gitignore&tests=.npmrc&tests=CHANGELOG.md&tests=LICENSE.md&tests=README.md&tests=simplify_sparql_results_doc_example.rq&tests=wikibase.png&tests=wikidata.jpg&tests=general_helpers.md&tests=get_entities.md&tests=get_entities_reverse_claims.md&tests=get_revisions.md&tests=install.md&tests=search_entities.md&tests=simplify_claims.md&tests=simplify_entities_data.md&tests=simplify_sparql_results.md&tests=sparql_query.md&tests=helpers.js&tests=parse_claim.js&tests=parse_responses.js&tests=rank.js&tests=simplify.js&tests=simplify_claims.js&tests=simplify_entity.js&tests=simplify_forms.js&tests=simplify_senses.js&tests=simplify_sitelinks.js&tests=simplify_sparql_results.js&tests=simplify_text_attributes.js&tests=sitelinks.js&tests=sitelinks_languages.js&tests=validate.js&tests=wikibase_time_to_date_object.js&tests=get_entities.js&tests=get_entities_from_sitelinks.js&tests=get_entity_revision.js&tests=get_many_entities.js&tests=get_reverse_claims.js&tests=get_revisions.js&tests=search_entities.js&tests=sparql_query.js&tests=build_url.js&tests=querystring_lite.js&tests=utils.js&tests=wikibase-sdk.js&tests=package.json&tests=build&tests=check_supported_datatypes&tests=compare_datatypes&tests=minify&tests=postpublish&tests=all_properties_datatypes.rq&tests=generate_sitelinks_languages.js&tests=update_sitelinks_languages&tests=update_dist&tests=update_toc&tests=update_wikidata_sdk&tests=L525.json&tests=P8098.json&tests=Q1.json&tests=Q19180293.json&tests=Q2112.json&tests=Q217447.json&tests=Q22002395.json&tests=Q271094.json&tests=Q275937.json&tests=Q328212.json&tests=Q4115189.json&tests=Q4132785.json&tests=Q571.json&tests=Q646148.json&tests=Q970917.json&tests=empty_values.json&tests=lexeme_claim.json&tests=multi_vars_sparql_results.json&tests=no_datatype_sparql_results.json&tests=old_claim_format.json&tests=properties_list.json&tests=results_with_labels_descriptions_and_aliases.rq&tests=results_with_labels_descriptions_and_aliases.json&tests=single_var_sparql_results.json&tests=sparql_results_with_optional_values.json&tests=sparql_results_with_statements.json&tests=wbgetentities_response.json&tests=general.js&tests=get_entities.js&tests=get_entities_from_sitelinks.js&tests=get_entity_revision.js&tests=get_many_entities.js&tests=get_reverse_claims.js&tests=get_revisions.js&tests=helpers.js&tests=tests_env.js&tests=utils.js&tests=parse.js&tests=querystring_lite.js&tests=rank.js&tests=search_entities.js&tests=simplify_claims.js&tests=simplify_claims_legacy_interface.js&tests=simplify_entity.js&tests=simplify_forms.js&tests=simplify_qualifiers.js&tests=simplify_senses.js&tests=simplify_sitelinks.js&tests=simplify_sparql_results.js&tests=simplify_text_attributes.js&tests=sitelinks_helpers.js&tests=sparql_query.js&tests=utils.js&tests=.npmrc&tests=README.md&tests=package.json&tests=wikidata-sdk.browser-global.js&tests=wikidata-sdk.js&tests=).%0A

9 last version(s):
packages/packages/w/wikibase-sdk.json: run /usr/bin/git checkout v7.6.1 -f
- 7.6.1: 4 file(s) matched :heavy_check_mark:
packages/packages/w/wikibase-sdk.json: run /usr/bin/git checkout v7.6.0 -f
- 7.6.0: 4 file(s) matched :heavy_check_mark:
packages/packages/w/wikibase-sdk.json: run /usr/bin/git checkout v7.5.0 -f
- 7.5.0: 4 file(s) matched :heavy_check_mark:
packages/packages/w/wikibase-sdk.json: run /usr/bin/git checkout v7.4.6 -f
- 7.4.6: 4 file(s) matched :heavy_check_mark:
packages/packages/w/wikibase-sdk.json: run /usr/bin/git checkout v7.4.5 -f
- 7.4.5: 4 file(s) matched :heavy_check_mark:
packages/packages/w/wikibase-sdk.json: run /usr/bin/git checkout v7.4.4 -f
packages/packages/w/wikibase-sdk.json: match *.js in /tmp/git013345953/dist but doesn't exists
- 7.4.4: 0 file(s) matched :heavy_exclamation_mark:
packages/packages/w/wikibase-sdk.json: run /usr/bin/git checkout v7.4.3 -f
packages/packages/w/wikibase-sdk.json: match *.js in /tmp/git013345953/dist but doesn't exists
- 7.4.3: 0 file(s) matched :heavy_exclamation_mark:
packages/packages/w/wikibase-sdk.json: run /usr/bin/git checkout v7.4.2 -f
packages/packages/w/wikibase-sdk.json: match *.js in /tmp/git013345953/dist but doesn't exists
- 7.4.2: 0 file(s) matched :heavy_exclamation_mark:
packages/packages/w/wikibase-sdk.json: run /usr/bin/git checkout v7.4.1 -f
packages/packages/w/wikibase-sdk.json: match *.js in /tmp/git013345953/dist but doesn't exists
- 7.4.1: 0 file(s) matched :heavy_exclamation_mark:
```

**Comment by klausenbusk** [comment]
PR opened: https://github.com/cdnjs/tools/pull/93
